pkgvars

```diff
@@ -1,6 +1,6 @@
 pkgname=citus
 pkgdesc='Citus (Open-Source)'
-pkglatest=9.4.5.citus-2
-releasepg=11,12
-nightlypg=11,12
+pkglatest=9.5.6.citus-1
+releasepg=11,12,13
+nightlypg=12,13
```

Comment:
Updating nightly PG versions do not have any effect. However they are harmless so we can let this one pass, because it makes sense for the script to set it to `12,13` here.